hnn_core/gui/gui.py

```diff
@@ -398,7 +421,102 @@ def compose(self):
                                     self.widget_tstop, self._load_info,
                                     self.layout)
 
-        return hnn_gui
+        if not return_layout:
+            return
+        else:
+            return self.app_layout
+
+    def show(self):
+        display(self.app_layout)
+
+    def take_screenshot(self, width=None, height=None, filename=None,
```

Comment:
```suggestion
    def show(self, freeze=True,
```

? Now that you have it working, it's helpful to think of the API a bit. Screenshot usually means it is not interactive ...

Reply:
It's indeed not interactive, by which I mean it will not influence the data model in the kernel. Probably `capture` could be a better name?

Reply:
Use `capture` now